website/docs/Examples.md

```diff
@@ -75,6 +75,7 @@ Links to notebook examples:
 1. **Long Context Handling**
 
    - Conversations with Chat History Compression Enabled - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_compression.ipynb)
+   - Long Context Handling as A Capability - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_capability_long_context_handling.ipynb)
```

Comment:
Do we want to remove the compressible agent notebook link if that's not working very well?

Reply:
removed.